src/appengine/private/components/upload-testcase/upload-form-simplified.html

```diff
@@ -146,11 +146,11 @@
         </template>
       </div>
       <div class="warning">
-        <b>WARNING</b>: ClusterFuzz doesn't support untrusted workloads.<br>
-        Only upload testcases that you trust enough to run on your own
-        machine. Note that the following jobs do use the Chrome sandbox
-        but this is not a guarantee of protection:<br>
-        [[formatSandboxedJobs(fieldValues.sandboxedJobs)]]
+        <b>WARNING</b>: ClusterFuzz only has partial support for untrusted workloads.<br>
+        ClusterFuzz only supports untrusted workloads on Linux jobs except for these:<br>
```

Comment:
maybe:

ClusterFuzz only supports untrusted workloads on Linux jobs, with the exception of these: